docker/Dockerfile.ci_gpu

```diff
@@ -1,9 +1,11 @@
 # CI docker GPU env
-FROM nvidia/cuda:10.1-cudnn7-devel-ubuntu18.04
+FROM nvidia/cuda:10.2-cudnn7-devel-ubuntu18.04
 
 ENV TZ=US
 RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
 
+RUN apt-key del 3bf863cc
```

Comment:
this looks a bit weird. could you add a comment to clarify it? It aims to fix nv image issue?

Reply:
Added a comment.